src/components/Header/FavoritesLink.tsx

```diff
@@ -6,7 +6,7 @@ import ChromeLink from '../ChromeLink';
 
 const FavoritesLink = () => {
   return (
-    <Button className="chr-c-button-masthead disabled" component={(props) => <ChromeLink {...props} href="#" />}>
+    <Button className="chr-c-button-masthead" component={(props) => <ChromeLink {...props} href="/FavoritedServices" />}>
```

Comment:
Can we disable the link for now? Until the backend is hooked up no favoriting will work anyways. And we should also use the `isDisabled` prop on the button now that the link has a pathname.